.github/ISSUE_TEMPLATE/feature---enhancement-request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature / Enhancement request
+about: Describe what you think is missing or could be improved in OpenRA.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- This is a guideline that shall help you to describe your idea for a missing feature or enhancement in order to allow evaluating it. Please follow the instructions and replace any placeholders that are written in capital letters. Instructions like this comment will not be visible in your report. -->
```

Comment:
"in order to allow evaluating it" doesn't sound correct. IMO it would be fine to end the sentence after "enhancement".